### DISCUSSION TITLE
Possibility to follow / star lists created by others?

### DISCUSSION THREAD

**Discussion by voxpelli** [community/community#20926] [Other Feature Feedback, Questions, & Ideas]
Would love to be able to follow lists from great curators and essentially be able to get an improved version of the `awesome-` collections.

Could be nice to even be able to add starred/ followed lists to one's own lists, so one can have a list of lists.

(Only main feature missing from lists then that the `awesome-` repos has is the ability to add a note on why a specific item is added to the list, as that's not always self explanatory)

(If all of that gets implemented, then next step would be to be able to generate the Markdown in an `awesome-` repo from a list and maybe even have accepted PR:s that adds to it be fed back into ones lists, adding the additions from the PR:s)

**Comment by Josephglz** [comment]
Its a great improvement.
